pkg/metrics/metrics.go

```diff
@@ -27,30 +27,37 @@ var (
 			Help: "A count of total failed requests received by the OCM Agent service",
 		}, []string{})
 
-	metricRequestFailure = prometheus.NewGaugeVec(
+	MetricRequestFailure = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "ocm_agent_request_failure",
 			Help: "Indicates that OCM Agent could not successfully process a request",
 		}, []string{"path"})
 
-	metricResponseFailure = prometheus.NewGaugeVec(
+	MetricResponseFailure = prometheus.NewGaugeVec(
```

Comment:
just checking if this var name needs to be capitalized, since there are other var names that are not like `metricServiceLogSent` or `metricRequestsTotal`